no-pdi/cmake/FindPDI.cmake

```diff
@@ -0,0 +1,45 @@
+include(CMakePrintHelpers)
+
+# Set the default search paths
+set(PDI_DIRS "${CMAKE_CURRENT_LIST_DIR}/..")
```

Comment:
As it is, this does not seem to be used.

But I'd use this rather then the `NO_PDI_PATH` below